src/cmd-buildupload

```diff
@@ -53,32 +64,33 @@ def parse_args():
 
 
 def cmd_upload_s3(args):
+    bucket, prefix = args.url.split('/', 1)
```

Comment:
Let's add the `bucket` and `prefix` to the `args` object itself so it naturally gets passed through to the other functions?

Reply:
Right, I'll add these as new arguments - and drop url once RHCOS pipeline stops using it

Reply:
I meant doing something like:

```
args.bucket, args.prefix = args.url.split('/', 1)
```

Would be nice to have it at the CLI level, though I don't think it's worth breaking compatibility.

Reply:
Implemented with additional optional params (`--bucket` and `--prefix`) which override `--url` if set. Once FCOS and RHCOS pipeline stops using `--url` we can remove that part.

Does that sound good?